### ISSUE TITLE
Follow up to #188: Shorten the Contact Us section

### ISSUE THREAD

**Issue by reece** [HGVSnomenclature/hgvs-nomenclature#194]
#188 was merged (by me) believing that the HVNC had already vetted the text. Commentary on that merge suggests otherwise. From the last comments on that PR: 

[@ifokkema wrote](https://github.com/HGVSnomenclature/hgvs-nomenclature/pull/188#issuecomment-2454456116):
> This also adds a reference to a method to ask us a question on GitHub:
> 
> > You may also submit questions to our
> > [GitHub repository Q&A board](https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/new?category=q-a).
> 
> Is it a good idea to offer multiple methods for this? Is one preferred over the other? Does the deadline that we set for ourselves also count for the GitHub method of asking us a question?

[@marinadistefano wrote](https://github.com/HGVSnomenclature/hgvs-nomenclature/pull/188#issuecomment-2455171913):
> Yeah, I do have concerns about missing something that it on our GitHub Q&A board instead of via email.

[@reece wrote](https://github.com/HGVSnomenclature/hgvs-nomenclature/pull/188#issuecomment-2455338837):
> If it's still up for discussion, then 1) I agree that we should drop the Q&A board, 2) I think the text about the mailing list usage, turn around time, etc. is far too long. (Preview at https://hgvs-nomenclature.org/main/).
> 
> I recommend:
> 
> - merging the first two paragraphs ("Join the..." and "Messages sent to...").
> - dropping the entire "When members of the community..." paragraph
> - dropping the Q&A board paragraph
> 
> Also, in a separate PR, I proposed to rename this section and add a new announcement-only group. See https://hgvs-nomenclature--191.org.readthedocs.build/191/.

Since that PR is merged, let's @ahwagner @marinadistefano @ifokkema continue the conversation here.

**Comment by marinadistefano** [comment]
Yes, there was a desire to be more transparent in the way the we respond to communications with the community so I drafted that text. I do think it's important that the community know how long it will generally take us to respond as many contact us to seek advice on actual clinical cases.

**Comment by ahwagner** [comment]
Sorry for any confusion here; I am responsible for adding the Q&A board text. I thought it was an obvious enhancement since we already provide that as an alternate community forum for GitHub-familiar people to ask and answer questions. This was added during my efforts to restructure this section to incorporate the text that @marinadistefano put together. I considered this a minor change and so only requested approval from @marinadistefano (as the task lead) to view the restructured section and text integration in #188. I am okay with removing the Q&A board text since this is apparently unpopular, and similarly fine with merging the first two paragraphs. However, I do not think we should remove the HVNC-vetted text that @marinadistefano contributed without committee review.
